examples/js/cameras/CombinedCamera.js

```diff
@@ -1,31 +1,38 @@
 /**
  *	@author zz85 / http://twitter.com/blurspline / http://www.lab4games.net/zz85/blog
  *
- *	A general perpose camera, for setting FOV, Lens Focal Length,
+ *		A general perpose camera, for setting FOV, Lens Focal Length,
  *		and switching between perspective and orthographic views easily.
  *		Use this only if you do not wish to manage
  *		both a Orthographic and Perspective Camera
  *
  */
 
 
-THREE.CombinedCamera = function ( width, height, fov, near, far, orthoNear, orthoFar ) {
+THREE.CombinedCamera = function ( width, height, fov, near, far, hyperfocusOffset, hyperfocusScale ) {
 
 	THREE.Camera.call( this );
-
+	// perspective
 	this.fov = fov;
-
+	this.far = far;
+	this.near = near;
+	//orthographic
 	this.left = - width / 2;
 	this.right = width / 2;
 	this.top = height / 2;
 	this.bottom = - height / 2;
 
+	this.aspect =  width / height;
+
 	// We could also handle the projectionMatrix internally, but just wanted to test nested camera objects
 
-	this.cameraO = new THREE.OrthographicCamera( width / - 2, width / 2, height / 2, height / - 2, 	orthoNear, orthoFar );
-	this.cameraP = new THREE.PerspectiveCamera( fov, width / height, near, far );
+	this.cameraO = new THREE.OrthographicCamera( this.left, this.right, this.top, this.bottom, this.near, this.far );
+	this.cameraP = new THREE.PerspectiveCamera( this.fov, this.aspect, this.near, this.far );
 
 	this.zoom = 1;
```

Comment:
Please move all setting of member variables before setting this.cameraO and this.cameraP - it would be cleaner this way